dietpi/dietpi-update

```diff
@@ -411,12 +405,7 @@ Do you wish to continue and update DietPi to v$COREVERSION_SERVER.$SUBVERSION_SE
 	Apply_1st_Run_Update_Sucess(){
 
 		# - 1st run setup
-		if (( $G_DIETPI_INSTALL_STAGE == 0 )); then
-
-			export G_DIETPI_INSTALL_STAGE=1
```

Comment:
NB: `export` and `G_DIETPI_INSTALL_STAGE=1` has no effect here. `$G_DIETPI_INSTALL_STAGE` is not used again until the script exits and `export` does not work in parent direction.

Reply:
@MichaIng 

Yep good spot, I probably added it in for consistency. But if not used, save some lines 😃